package.json

```diff
@@ -14,7 +14,7 @@
     "test": "jest",
     "eslint": "eslint --ext .tsx,.js ./src/",
     "lint": "npm run eslint",
```

Comment:
Is there a `license-check-and-add check` or something similar that could be added to the `lint` script as well so that it fails if any files are missing their headers?